.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x]
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Setup env
+      run:
+        echo ::set-env name=GOFLAGS::-mod=vendor
```

Comment:
Would you be willing to change this (and the same thing under `name: Vet` below) to look either like...

```
      run: 'echo ::set-env name=GOFLAGS::-mod=vendor'
```

...or...

```
      run: |
        echo ::set-env name=GOFLAGS::-mod=vendor
```

...?  I'm afraid if someone needs to add a second command to run, they'll add it on a new line under the existing command, and then the two lines will just get squashed together into one line instead of being treated as separate lines.

Reply:
👍